frontend/app/helpers/get-cookie.js

```diff
@@ -0,0 +1,10 @@
+import { helper } from "@ember/component/helper";
+
+export function getCookie() {
+  let cookieArray = document.cookie.valueOf().split(";");
+  let cookie = cookieArray[cookieArray.length - 1];
```

Comment:
I think we should probably add a regex here to make sure we get the correct cookie instead of getting it based on the position which is definetly not fixed.